evaluators/jsonpath-evaluator/features/etc/feature.properties

```diff
@@ -0,0 +1,201 @@
+################################################################################
+#  Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Is the year mandatory? I actually copied the properties file from an existing feature. Can add if required though :)